truss/truss_handle.py

```diff
@@ -244,7 +244,13 @@ def update_requirements_from_file(self, requirements_filepath: str):
         at the given path.
         """
         with Path(requirements_filepath).open() as req_file:
-            self.update_requirements([line.strip() for line in req_file.readlines()])
+            self.update_requirements(
+                [
+                    line.strip()
+                    for line in req_file.readlines()
+                    if not line.startswith("#")
```

Comment:
Do you want to do line.strip().startswith('#')?

Reply:
Good catch.